### DISCUSSION TITLE
关于不常见的词组或容易有歧义的翻译内容时，是否在文档第一次出现时，带上英文原文

### DISCUSSION THREAD

**Discussion by HunDun0Ben** [reactjs/zh-hans.react.dev#1628] [General]
例如脱围机制，这个词在日常中并不常见。
如果像是 Linting 章节一样， 第一次出现时带上英文原文，是否更方便的理解和联系。
![example](https://github.com/user-attachments/assets/84b0a304-3377-41d5-aaa1-f4d1d357309f)

**Comment by loveloki** [comment]
言之有理，欢迎提交 PR 进行修改

**Reply by HunDun0Ben** [reply]
谢谢回复. 
已经提交了 PR. 第一次尝试提交 PR, 如果有问题的话请谅解. 谢谢.

**Reply by loveloki** [reply]
不要害怕犯错，有 reviewer 会帮忙看。

不同项目可能有不同的要求，修改前确认下项目的注意事项就行了。以及提交后再确认一下修改内容。

其他方面也是如此，多动手练习，熟能生巧。

祝编程愉快！

**Comment by HunDun0Ben** [comment]
好的. 我可能需要花一些时间去熟悉, 和了解提交的流程.

**Comment by loveloki** [comment]
已经更新 Wiki，感谢指出
